app/charges/controllers.py

```diff
@@ -279,3 +279,52 @@ def edit_charge(user, user_data):
     except Exception as e:
         db.session.rollback()
         emit("edit_charge", Response.EditError)
+
+##
+## @brief       closes a charge
+##
+## A charge can only be closed by an Admin user, if another user tries to close the 
+## charge, then this controller will send an email and an in app notification to the 
+## committee-head requesting for them to close the charge.
+##
+## @param      user_data  The user data to request a charge closing
+##
+##             - token (required): Token of user.
+##             - charge (requred): charge to close
+##
+## @return     { description_of_the_return_value }
+##
+@socketio.on('close_charge')
+@ensure_dict
+@get_user
+def close_charge(user, user_data):
+    charge = Charges.query.filter_by(id = user_data.get("charge",-1)).first()
+
+    if charge is None or user is None:
+        emit("close_charge", Response.UsrChargeDontExist)
+        return
+
+    committee = Committees.query.filter_by(id = user_data.get("committee_id",-1)).first()
+    membership = committee.members.filter_by(member= user).first()
+
+    # User is NOT admin
+    if (membership is None or membership.role != Roles.CommitteeHead) and not user.is_admin:
```

Comment:
Checked in with Bryan and only `CommitteeHead`s should be able to send a close request. The workflow for having a charge closed is:

1. Committee Head requests to close a charge
2. Admin receives email saying Committee Head wants to close a charge
3. Admin closes the charge

So the logic here needs to be changed so that only `CommitteeHead`s can send close requests.